var/spack/repos/builtin/packages/py-abipy/package.py

```diff
@@ -13,44 +13,65 @@ class PyAbipy(PythonPackage):
     homepage = "https://github.com/abinit/abipy"
     pypi = "abipy/abipy-0.2.0.tar.gz"
 
+    version("0.9.3", sha256="2f750c1ddcf78d0ec77c1bd44624ff96f80436307237589c02da127106514b75")
     version("0.2.0", sha256="c72b796ba0f9ea4299eac3085bede092d2652e9e5e8074d3badd19ef7b600792")
 
     variant("gui", default=False, description="Build the GUI")
-    variant("ipython", default=False, description="Build IPython support")
-
-    depends_on("python@2.7:")
+    variant("ipython", default=False, when="0.2.0", description="Build IPython support")
 
     depends_on("py-setuptools", type="build")
     depends_on("py-cython", type="build")
```

Comment:
Is cython actually used?

Reply:
I just tested and in also builds finde without it (also the old version), so I have removed it now.